lib/moab/storage_object_validator.rb

```diff
@@ -92,14 +95,32 @@ def check_correctly_formed_moabs
         errors.concat check_sub_dirs(version_sub_dirs, version_dir, EXPECTED_VERSION_SUB_DIRS)
         after_result_size = errors.size
         # run the following checks if this version dir passes check_sub_dirs, even if some prior version dirs didn't
+        errors.concat check_required_manifest_files(version_path, version_dir) if before_result_size == after_result_size
         if before_result_size == after_result_size
-          data_dir_path = "#{version_path}/#{DATA_DIR_NAME}"
-          data_sub_dirs = sub_dirs(data_dir_path)
-          errors.concat check_sub_dirs(data_sub_dirs, version_dir, EXPECTED_DATA_SUB_DIRS)
-          errors.concat check_required_manifest_files(version_path, version_dir)
+          errors.concat check_expected_data_sub_dirs(version_path, version_dir)
+          if after_result_size == errors.size
+            errors.concat check_metadata_dir_file_only(version_path)
```

Comment:
well, shoot.  `check_metadata_dir_fileSSSSSSS_only`. (do you speak parseltongue?)

Reply:
hahaha 👍